requirements.in

```diff
@@ -22,7 +22,7 @@
 ##########################################
 grpcio-tools==1.57.0
```

Comment:
grpcio-tools shall be align with grpcio itself to avoid incompatibility issues.
https://pypi.org/project/grpcio-tools/

Reply:
This is a bit problematic as grpcio-tools are defined in template and grpcio in SDK.
Why is this at all?
Could we move grpcio-tools to SDK, too? (Is it directly used in template?)

Reply:
Do we need them at all in the template? As far as I see, the tools provide the compiler, but we only compile the protofiles in the sdk...